### ISSUE TITLE
Banner/header comment?

### ISSUE THREAD

**Issue by stevenvachon** [hxlniada/webpack-concat-plugin#65]
Would be nice to have this, similar [`webpack.BannerPlugin`](https://webpack.js.org/plugins/banner-plugin/).

**Comment by MuhammadDicky** [comment]
Yeah, webpack banner plugin not process file beside from entry point even you set entryOnly to false.
